db/seeds/development.rb

```diff
@@ -27,6 +27,12 @@
 # See doc/COPYRIGHT.rdoc for more details.
 #++
 
+# Disable mail delivery for the duration of this task
+ActionMailer::Base.perform_deliveries = false
+
+# Avoid asynchronous DeliverWorkPackageCreatedJob
+Delayed::Worker.delay_jobs = false
```

Comment:
Isn't that quite harsh?

I am not sure what we are currently doing in delayed jobs, but it seems that this can cause quite some collateral damage.

Imagine we wanted to seed repositories. Not executing delayed jobs there would probably break them, wouldn't it?